File: uri/1 - Basics/1015/Main.rb
def main()
	l1=gets.split()
    l2=gets.split()
    x1,y1 = [l1[0].to_f, l1[1].to_f]
    x2,y2 = [l2[0].to_f, l2[1].to_f]

    print("%.4f\n" %  Math.sqrt( (x2-x1) ** 2 + (y2-y1) ** 2 ) )
end

main()

File: uri/1 - Basics/1015/Main.cs
using System;

class URI {
	static void Main(string[] args) {
		var l1 = Array.ConvertAll(Console.ReadLine().Split(" "),double.Parse);
		var l2 = Array.ConvertAll(Console.ReadLine().Split(" "),double.Parse);
		double x1 = l1[0];
	    double y1 = l1[1];
	    double x2 = l2[0];
	    double y2 = l2[1];

		double distance = Math.Sqrt(Math.Pow( (x2-x1),2) + Math.Pow( (y2-y1),2));
		Console.WriteLine($"{distance:F4}");
	}	
}
